From 3dc6d5f32b07baf5f2344c667be590551b337350 Mon Sep 17 00:00:00 2001
From: Yuri Chornoivan <yurchor@ukr.net>
Date: Fri, 8 Mar 2019 15:02:35 +0200
Subject: Add necessary files to use native Indonesian language for all
 Indonesian docs.

Please translate the entities appropriately. Thanks.

CCMAIL: wantoyek@gmail.com
---
 .../id/entities/ccbysa4-notice.docbook             |  4 ++
 src/customization/id/entities/underCCBYSA4.docbook |  3 ++
 src/customization/id/entities/underLGPL.docbook    |  2 +
 src/customization/id/lang.entities                 | 41 +++++++++--------
 src/customization/id/user.entities                 | 51 ++++++++++++++++++++++
 src/customization/xsl/id.xml                       | 10 +++++
 6 files changed, 93 insertions(+), 18 deletions(-)
 create mode 100644 src/customization/id/entities/ccbysa4-notice.docbook
 create mode 100644 src/customization/id/entities/underCCBYSA4.docbook
 create mode 100644 src/customization/id/entities/underLGPL.docbook
 create mode 100644 src/customization/xsl/id.xml

diff --git a/src/customization/id/entities/ccbysa4-notice.docbook b/src/customization/id/entities/ccbysa4-notice.docbook
new file mode 100644
index 0000000..d586e17
--- /dev/null
+++ b/src/customization/id/entities/ccbysa4-notice.docbook
@@ -0,0 +1,4 @@
+<para>Permission is granted to copy, distribute and/or modify this
+document under the terms of the Creative Commons
+Attribution-ShareAlike 4.0 International or any later.
+A copy of the license is included in <xref linkend="cc-by-sa-4"/>.</para>
diff --git a/src/customization/id/entities/underCCBYSA4.docbook b/src/customization/id/entities/underCCBYSA4.docbook
new file mode 100644
index 0000000..f0cd938
--- /dev/null
+++ b/src/customization/id/entities/underCCBYSA4.docbook
@@ -0,0 +1,3 @@
+<para id="cc-by-sa-4" xreflabel="the credits and license section">This documentation is licensed
+under the terms of the <ulink type="commondoc" url="ccbysa4-license.html">Creative
+Commons Attribution-ShareAlike 4.0 International</ulink>.</para>
diff --git a/src/customization/id/entities/underLGPL.docbook b/src/customization/id/entities/underLGPL.docbook
new file mode 100644
index 0000000..5de95d5
--- /dev/null
+++ b/src/customization/id/entities/underLGPL.docbook
@@ -0,0 +1,2 @@
+<para>This program is licensed under the terms of the <ulink
+type="commondoc" url="lgpl-license.html">GNU Lesser General Public License</ulink>.</para>
diff --git a/src/customization/id/lang.entities b/src/customization/id/lang.entities
index ec3dccc..c584adc 100644
--- a/src/customization/id/lang.entities
+++ b/src/customization/id/lang.entities
@@ -1,3 +1,5 @@
+<!-- Do NOT change this file: it provides important defaults -->
+<!-- test -->
 <!ENTITY language "id">
 
 <!-- These entities should be translated, should therefore be stored
@@ -10,39 +12,42 @@
      text                                                          -->
 
 <!-- Licence links -->
-<!ENTITY underGPL           PUBLIC "-//KDE//DOCUMENT GPL Licence Declaration//ID"
+<!ENTITY underGPL           PUBLIC "-//KDE//DOCUMENT GPL Licence Declaration//EN"
   "entities/underGPL.docbook"                       ><!-- level: para -->
-<!ENTITY underCCBYSA4       PUBLIC "-//KDE//DOCUMENT CC BY-SA 4.0 Licence Declaration//ID"
-  "../en/entities/underCCBYSA4.docbook"                   ><!-- level: para -->
-<!ENTITY underFDL           PUBLIC "-//KDE//DOCUMENT FDL Licence Declaration//ID"
+<!ENTITY underLGPL          PUBLIC "-//KDE//DOCUMENT LGPL Licence Declaration//EN"
+  "entities/underLGPL.docbook"                       ><!-- level: para -->
+<!ENTITY underCCBYSA4       PUBLIC "-//KDE//DOCUMENT CC BY-SA 4.0 Licence Declaration//EN"
+  "entities/underCCBYSA4.docbook"                   ><!-- level: para -->
+<!ENTITY underFDL           PUBLIC "-//KDE//DOCUMENT FDL Licence Declaration//EN"
   "entities/underFDL.docbook"                       ><!-- level: para -->
-<!ENTITY underBSDLicense    PUBLIC "-//KDE//DOCUMENT BSD Licence Declaration//ID"
+<!ENTITY underBSDLicense    PUBLIC "-//KDE//DOCUMENT BSD Licence Declaration//EN"
   "entities/underBSDLicense.docbook"                ><!-- level: para -->
-<!ENTITY underArtisticLicense PUBLIC "-//KDE//DOCUMENT Artistic Licence Declaration//ID"
+<!ENTITY underArtisticLicense PUBLIC "-//KDE//DOCUMENT Artistic Licence Declaration//EN"
   "entities/underArtisticLicense.docbook"           ><!-- level: para -->
-<!ENTITY underX11License    PUBLIC "-//KDE//DOCUMENT X11 Licence Declaration//ID"
+<!ENTITY underX11License    PUBLIC "-//KDE//DOCUMENT X11 Licence Declaration//EN"
   "entities/underX11License.docbook"                ><!-- level: para -->
 
-<!ENTITY reporting.bugs     PUBLIC "-//KDE//DOCUMENT Report Bugs//ID"
+<!ENTITY reporting.bugs     PUBLIC "-//KDE//DOCUMENT Report Bugs//EN"
   "entities/report-bugs.docbook"                       ><!-- level: ? -->
-<!ENTITY updating.documentation PUBLIC "-//KDE//DOCUMENT Updating Documentation//ID"
+<!ENTITY updating.documentation PUBLIC "-//KDE//DOCUMENT Updating Documentation//EN"
   "entities/update-doc.docbook"                     ><!-- level: para -->
-<!ENTITY help.menu.documentation PUBLIC "-//KDE//DOCUMENT Help Menu Documentation//ID"
+<!ENTITY help.menu.documentation PUBLIC "-//KDE//DOCUMENT Help Menu Documentation//EN"
   "entities/help-menu.docbook"                      ><!-- level: variablelist -->
-<!ENTITY install.intro.documentation PUBLIC "-//KDE//DOCUMENT Installation General Information//ID"
+
+<!-- Entities install.intro.documentation + install.compile.documentation
+     are deprecated and should be removed in Frameworks 6 -->
+
+<!ENTITY install.intro.documentation PUBLIC "-//KDE//DOCUMENT Installation General Information//EN"
   "entities/install-intro.docbook"                     ><!-- level: para -->
-<!ENTITY install.compile.documentation PUBLIC "-//KDE//DOCUMENT Compilation Instructions//ID"
+<!ENTITY install.compile.documentation PUBLIC "-//KDE//DOCUMENT Compilation Instructions//EN"
   "entities/install-compile.docbook"                     ><!-- level: para -->
 
 
 <!-- CC BY-SA 4 notice -->
-<!-- In order to translate it, copy it into <lang>/entities/ccbysa4-notice.docbook
-and change the reference from English version to the translated document
-and remove this comment. -->
-<!ENTITY CCBYSA4Notice PUBLIC "-//KDE//DOCUMENT CC BY-SA 4 Documentation Notice//ID"
-         "../en/entities/ccbysa4-notice.docbook">
+<!ENTITY CCBYSA4Notice PUBLIC "-//KDE//DOCUMENT CC BY-SA 4 Documentation Notice//EN"
+         "entities/ccbysa4-notice.docbook">
 <!-- FDL notice -->
-<!ENTITY FDLNotice PUBLIC "-//KDE//DOCUMENT GNU Free Documentation Notice//ID"
+<!ENTITY FDLNotice PUBLIC "-//KDE//DOCUMENT GNU Free Documentation Notice//EN"
          "entities/fdl-notice.docbook">
 <!-- meant to be included, so no NDATA or CDATA (why?) -->
 
diff --git a/src/customization/id/user.entities b/src/customization/id/user.entities
index 5f54ec6..1ea8228 100644
--- a/src/customization/id/user.entities
+++ b/src/customization/id/user.entities
@@ -6,3 +6,54 @@
      names (which if they occur will cost you a _lot_ of time)
  -->
 
+<!ENTITY Alt	"<keycap>Alt</keycap>">
+<!ENTITY applications	"<productname>Applications</productname>">
+<!ENTITY Backspace	"<keysym>Backspace</keysym>">
+<!ENTITY cdrom	'<hardware>CD-ROM drive</hardware>'>
+<!ENTITY Ctrl	"<keycap>Ctrl</keycap>">
+<!ENTITY Del	"<keycap>Del</keycap>">
+<!ENTITY Down	"<keysym>Down</keysym>"> <!-- Cursor key-->
+<!ENTITY dpi	'<acronym>dpi</acronym>'>
+<!ENTITY eg     "<abbrev>e.g.</abbrev>">
+<!ENTITY End	"<keycap>End</keycap>">
+<!ENTITY Enter	"<keysym>Enter</keysym>">
+<!ENTITY Esc	"<keycap>Esc</keycap>">
+<!ENTITY etc    "<abbrev>etc.</abbrev>">
+<!ENTITY FAQ	'<acronym>FAQ</acronym>'>
+<!ENTITY Home	"<keycap>Home</keycap>">
+<!ENTITY ie     "<abbrev>i.e.</abbrev>">
+<!ENTITY infocenter	"<application>Info Center</application>">
+<!ENTITY Ins	"<keycap>Ins</keycap>">
+<!ENTITY kde-frameworks	"<productname>&kde; Frameworks</productname>">
+<!ENTITY kf5-full	"&kde-frameworks; <productnumber>5</productnumber>">
+<!ENTITY Left	"<keysym>Left</keysym>"> <!-- Cursor key-->
+<!ENTITY LMB    "<mousebutton>left</mousebutton> mouse button">
+<!ENTITY Meta	"<keysym>Meta</keysym>"> <!-- Super key or Windows key -->
+<!ENTITY MMB    "<mousebutton>middle</mousebutton> mouse button">
+<!ENTITY OS	"operating system"><!-- <acronym>OS</acronym>-->
+<!ENTITY partman "<application>KDE Partition Manager</application>">
+<!ENTITY plasma-mobile	"<productname>&kde; Plasma Mobile</productname>">
+<!ENTITY plasma-media-center	"<productname>&kde; Plasma Media Center</productname>">
+<!ENTITY plasma-netbook	"<productname>&kde; Plasma Netbook</productname>">
+<!ENTITY plasma-desktop	"<productname>&kde; Plasma Desktop</productname>">
+<!ENTITY plasma-workspaces	"<productname>&kde; Plasma Workspaces</productname>">
+<!ENTITY plasmapa       "<productname>&kde; Plasma Pulseaudio</productname>">
+<!ENTITY PgUp	"<keysym>PgUp</keysym>">
+<!ENTITY PgDn	"<keysym>PgDn</keysym>">
+<!ENTITY Right	"<keysym>Right</keysym>"> <!-- Cursor key-->
+<!ENTITY RMB    "<mousebutton>right</mousebutton> mouse button">
+<!ENTITY Shift	"<keycap>Shift</keycap>">
+<!ENTITY Space	"<keysym>Space</keysym>">
+<!ENTITY systemsettings	"<application>System Settings</application>">
+<!ENTITY systemtray "<application>system tray</application>">
+<!ENTITY Tab	"<keycap>Tab</keycap>">
+<!ENTITY Up	"<keysym>Up</keysym>"> <!-- Cursor key-->
+
+<!-- Deprecated, will be removed at a future time, do not use -->
+<!ENTITY kcontrolcenter	"<application>&kde; Control Center</application>">
+<!ENTITY kf5	"<acronym>KF5</acronym>">
+<!ENTITY kmenu  "<guimenu>K</guimenu> menu">
+<!ENTITY kde-sc	"&kde; <acronym>SC</acronym>">
+<!ENTITY kde-sc-full	"<productname>&kde; Software Compilation</productname>">
+<!ENTITY plasma-active	"<productname>&kde; Plasma Active</productname>">
+
diff --git a/src/customization/xsl/id.xml b/src/customization/xsl/id.xml
new file mode 100644
index 0000000..2206d53
--- /dev/null
+++ b/src/customization/xsl/id.xml
@@ -0,0 +1,10 @@
+<l:i18n xmlns:l="http://docbook.sourceforge.net/xmlns/l10n/1.0">
+<l:l10n language="id" english-language-name="Indonesian">
+
+   <l:gentext key="nav-home" text="Contents"/>
+
+   <l:gentext key="footer-doc-comment" text="Would you like to make a comment or contribute an update to this page?"/>
+   <l:gentext key="footer-doc-feedback" text="Send feedback to the "/>
+   <l:gentext key="footer-doc-teamname" text="KDE Docs Team"/>
+</l:l10n>
+</l:i18n>
-- 
cgit v1.1

